### PR TITLE
docs(pkg): readme info

### PR DESCRIPTION
**Pull request by ematipico**
**What kind of change does this PR introduce?**

Documentation. Part of #247

**Did you add tests for your changes?**
No
**If relevant, did you update the documentation?**
yes
**Summary**
Created read me file for info sub package

**Does this PR introduce a breaking change?**
No
**Other information**

### CODE REVIEW

File: packages/info/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+## Description
+
+It returns a set of information related to the local enviroment.
+
+## Installation
+
+```bash
+npm i -D @webpack-cli/info
+```
```
Comment (ev1stensberg) [inline packages/info/README.md]: Could you add an example to this as well, and how to use it?

**Comment by ev1stensberg** [comment]
@ematipico status on this?

**Comment by ematipico** [comment]
PTAL

File: packages/info/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# webpack-cli Info
```
Comment (ev1stensberg) [inline packages/info/README.md]: Small I and otherwise good to go, thanks @ematipico !

**Pull request merged**